src/main/java/com/epam/page/object/generator/adapter/classbuildable/FormClass.java

```diff
@@ -85,7 +85,9 @@ private String getAnnotationValue(Selector selector, String uniquenessValue,
         } else if (selector.isCss()) {
             return selectorUtils.resultCssSelector(selector, uniquenessValue, uniqueness);
         }
-        return null;
+        IllegalArgumentException e = new IllegalArgumentException(selector.toString());
```

Comment:
This exception is going to be printed to console 2 times. First when logger.error called, second when application crashed. To avoid this just do not log exception as soon as it have been created.